crates/virtio-queue/src/lib.rs

```diff
@@ -179,6 +182,12 @@ pub trait QueueStateT: for<'a> QueueStateGuard<'a> {
     /// Return the index of the next entry in the available ring.
     fn next_avail(&self) -> u16;
 
+    /// Return the index for the next descriptor in the used ring.
```

Comment:
Could we group getter and setter together by moving line 185-187 to line 190?

Reply:
Sure!

Reply:
Done.